app/main/views/briefs.py

```diff
@@ -417,7 +417,7 @@ def _render_not_eligible_for_brief_error_page(brief, clarification_question=Fals
             reason = data_reason_slug = "supplier-not-on-lot"
     else:
         reason = "supplier-not-on-framework"
-        data_reason_slug = "supplier-not-on-{}".format(brief['frameworkFramework'])
+        data_reason_slug = "supplier-not-on-{}".format(brief['frameworkSlug'])
```

Comment:
Does this mean we also need to add a case to the `not_is_supplier_eligible_for_brief_error` template to match the case when this is DOS2? Or will it "just work"?